### PR TITLE
Issue Template: Batch email request form

### LINKED ISSUE #448: Create an issue template for making requests to send an email blast
Create an issue template for requesting the CS team to send out email blasts

<ins>Content to include in template<ins>

This issue template can be used to request that the CS team send out an ad hoc email blast when Cal-ITP has noteworthy news to share. [The process for preparing an email blast is detailed in this document](https://docs.google.com/document/d/1FZBgW3vASHcPKmdD38V3zexcBNtyz-jZnjHQcwf4HBU/edit).

[Req. field] **Link to the email copy**:
_This should be a google doc link_

[Req. field] **Is the copy finalized?**:
- [ ] Yes
- [ ] No

_If no, the CS team will wait to take action on this issue until told that the copy is final._

[Req. fields] **Confirm that the linked document provides the following pieces of information**:
- [ ] Clear subject line 
- [ ] Pre-header (Optional, this is the line that shows up below the Subject Line in your mobile inbox. If your email has no pre-header, check this box anyway)
- [ ] A concise body 
- [ ] A signature at the end of the body (eg, From, The Cal-ITP Team)

[Req. field] **What email address should this be from?**:
- [ ] hello@calitp.org (preferred)
- [ ] calitp@dot.ca.gov
- [ ] Other

[Req. field] **Send Date**:

[Req. field] **Do you want to receive a test email?**:
- [ ] Yes
- [ ] No

**Please enter the names of anyone else who should receive a test email, if applicable:**

### PR DESCRIPTION
**Pull request by AnthonyRollins**
Closes #448 

## This PR
Adds a new issue template file for batch email requests from the CS team.

## Outstanding items
- [ ] Fix an issue with `checkboxes` type not allowing for `validations: required: True`
- [ ] Determine if there should be a question about the audience for the email blast

### CODE REVIEW

**Comment by AnthonyRollins** [comment]
@o-ram This is ready for your review.

File: .github/ISSUE_TEMPLATE/batch-email-requests.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    id: email-copy
+    attributes:
+      label: Provide a link to the email copy
+      description: The link should be a google doc
+      placeholder: ex https://docs.google.com/
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
```
Comment (o-ram) [inline .github/ISSUE_TEMPLATE/batch-email-requests.yml:21]: I think the question for this part is missing. It should ask the following question and be required to answer.

Is the copy finalized?:
- [ ] Yes
- [ ]  No

File: .github/ISSUE_TEMPLATE/batch-email-requests.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        _If no, the CS team will wait to take action on this issue until told that the copy is final_
+  - type: checkboxes
+    id: information
+    attributes:
+      label: Confirm that the linked document above provides the following pieces of information
+      description: Select all the apply.
```
Comment (o-ram) [inline .github/ISSUE_TEMPLATE/batch-email-requests.yml:27]: small typo, should say "Select all that apply"

File: .github/ISSUE_TEMPLATE/batch-email-requests.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        _If no, the CS team will wait to take action on this issue until told that the copy is final_
+  - type: checkboxes
+    id: information
+    attributes:
+      label: Confirm that the linked document above provides the following pieces of information
```
Comment (o-ram) [inline .github/ISSUE_TEMPLATE/batch-email-requests.yml:26]: Can we make this question required as well?

**Review by o-ram — CHANGES_REQUESTED** [review]
A few things I noticed on some questions, but otherwise it's looking good!

File: .github/ISSUE_TEMPLATE/batch-email-requests.yml (hunk 3 above)
Comment (o-ram) [reply .github/ISSUE_TEMPLATE/batch-email-requests.yml:30]: that's fine, not a huge deal! Should be important enough that people won't skip it.

**Comment by AnthonyRollins** [comment]
@o-ram I added in some changes, and still need to figure out how to make the `checkboxes` question required.

**Comment by AnthonyRollins** [comment]
@thekaveman Not sure what the issue is here, but I can't seem to figure out how to make `type: checkboxes` use a `validation` I took a look at the Gihub forums and found a small discussion on this topic: https://github.com/orgs/community/discussions/45084, but no one seems to have an answer on this.

**Comment by thekaveman** [comment]
> @thekaveman Not sure what the issue is here, but I can't seem to figure out how to make `type: checkboxes` use a `validation` I took a look at the Gihub forums and found a small discussion on this topic: https://github.com/orgs/community/discussions/45084, but no one seems to have an answer on this.

Yeah I'm not seeing much in the way of helpful information in that thread...

@angela-tran @machikoyasuda @lalver1 @allejo is this anything you all have seen / know how to address?

**Comment by lalver1** [comment]
I made a test issue template using `checkboxes` and `validations: required: true` and yeah, I see that the `checkboxes` section doesn't get the red asterisk that they are required. A workaround that may help is to mark each `option` with `required: true`. Sort of like:
```
 - type: checkboxes
    id: information
    attributes:
      label: Confirm that the linked document above provides the following pieces of information
      description: Select all that apply.
      options:
        - label: Clear subject line
          required: true
        - label: Pre-header (Optional, this is the line that shows up below the Subject Line in your mobile inbox. If your email has no pre-header, check this box anyway)
        - label: A concise body
        - label: A signature at the end of the body (eg, From, The Cal-ITP Team)
```

It looks like the red asterisk shows up on each `label` and the `Submit new issue` button behaves as expected:

image

**Comment by AnthonyRollins** [comment]
Thanks, @lalver1, for figuring out a workaround!

**Comment by AnthonyRollins** [comment]
@o-ram This is ready to publish.

**Review by o-ram — APPROVED** [review]
LGTM! Thanks, everyone!

**Pull request merged**